File: MediaData/src/main/java/com/xu/mediadata/entity/Folder.java
package com.xu.mediadatatest.entity;

/**
 * desc:
 * *
 * user: xujj
 * time: 2022/8/26 14:53
 **/
public class Folder {

    private String name;
    private String path;
    private int musicCount;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public int getMusicCount() {
        return musicCount;
    }

    public void setMusicCount(int musicCount) {
        this.musicCount = musicCount;
    }

    @Override
    public String toString() {
        return "Folder{" +
                "name='" + name + '\'' +
                ", path='" + path + '\'' +
                ", musicCount=" + musicCount +
                '}';
    }
}

File: MediaData/src/main/java/com/tool/mediadata/entity/Folder.kt
package com.tool.mediadata.entity

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

/**
 * desc:
 **
 * user: xujj
 * time: 2023/4/14 17:08
 **/
@Parcelize
data class Folder(
    var name: String,
    var path: String,
    var musicCount: Int
) : Parcelable {

    companion object {
        fun example(name: String = "xxx", path: String = "xxx"): Folder {
            return Folder(name, path, 0)
        }
    }
}